packages/opentelemetry-exporter-collector/test/certs/regenerate.sh

```diff
@@ -0,0 +1,11 @@
+rm ca.crt ca.key client.crt client.csr client.key server.crt server.csr server.key
```

Comment:
Can you please add some comments or documentation on what this file does, when it should be run, why it is here?

Reply:
sure, would [this document](https://github.com/open-telemetry/opentelemetry-js/blob/master/doc/development-guide.md) be a good place for it?

Reply:
Don't think so. That file is for general contribution guidelines and things that apply to the whole repository, not a specific package. I would just add it as comments to the top of this file, and/or add a note in the collector exporter readme.

Reply:
Got it, thanks.